src/Worker.Extensions.DurableTask/ManagedIdentityOptions.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask
```

Comment:
We should use file-scoped namespaces consistently in all our source files.
```suggestion
namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask;
```